internal/controller/nginx/config/servers_template.go

```diff
@@ -93,6 +93,12 @@ server {
         internal;
         {{ end }}
 
+        {{ if ne $l.MirrorSplitClientsVariableName "" -}}
+        if (${{ $l.MirrorSplitClientsVariableName }} = "") {
+            return 204;
```

Comment:
Just to confirm, there are no nginx logs when this occurs, correct? So we only see logs when the mirror actually sends?